src/customers/account-dashboard-approval-rules.md

```diff
@@ -19,6 +19,18 @@ _Company Approval Rules_
 
 Depending on the company role and permissions, users may have the option to edit, delete, or view approval rules.
 
+## Purchase order payment methods
+
+Purchase order approval supports both online and offline payment methods.
+
+All default offline payment methods are supported for purchase order approvals.
+For online payments, the following methods are supported:
+
+* Paypal Express
```

Comment:
@dobooth we should indicate somewhere that we only support `In-Context Checkout Experience`. out-of-context works mostly but there are still issues & we dont support it in POs yet

Reply:
Done